ArduPlane/Log.cpp

```diff
@@ -393,8 +393,14 @@ const struct LogStructure Plane::log_structure[] = {
       "QTUN", "QffffffeccfBB", "TimeUS,ThI,ABst,ThO,ThH,DAlt,Alt,BAlt,DCRt,CRt,TMix,Trn,Ast", "s----mmmnn---", "F----00000---" , true },
 #endif
 
-// @LoggerMessage: PIQR,PIQP,PIQY,PIQA
-// @Description: QuadPlane Proportional/Integral/Derivative gain values for Roll/Pitch/Yaw/Z
+// @LoggerMessage: PIQR
+// @Description: QuadPlane Proportional/Integral/Derivative gain values for Roll
+// @LoggerMessage: PIQP
+// @Description: QuadPlane Proportional/Integral/Derivative gain values for Pitch
+// @LoggerMessage: PIQY
+// @Description: QuadPlane Proportional/Integral/Derivative gain values for Yaw
+// @LoggerMessage: PIQA
+// @Description: QuadPlane Proportional/Integral/Derivative gain values for Altitude
```

Comment:
```suggestion
// @Description: QuadPlane Proportional/Integral/Derivative gain values for vertical acceleration
```

Reply:
Just to confirm, it is "vertical acceleration", right? (I was kind of expecting vertical velocity - but without good reason, as I don't know the code!)

Reply:
Correct, we do have a velocity PID too.